setup.py

```diff
@@ -116,8 +116,17 @@ def build_extensions(self):
                 # Support macports on Mac OS X.
                 self.compiler.add_include_dir('/opt/local/include')
```

Comment:
> ```python
> if self.use_system_http_parser:
> ```

Though llhttp hasn't hit Debian or HomeBrew repository, users could `make install` llhttp manually into their `/usr/local`. We should probably add a new option `use-system-llhttp` here too.